README.md

```diff
@@ -1 +1,1175 @@
+
+```bash
+   $ ballerina init
+```
+### Developing the SQL data
+
+Ballerina language has built-in support for writing web services. The `service` keyword in Ballerina simply defines a web service. Inside the service block, we can have all the required resources. You can define a resource inside the service. You can implement the business logic inside a resource using Ballerina language syntax.
+We can use the schema of the following database to store the student's data and student's results data.
+
+The StudentDetailsDB database is used to store the student's data in the StudentDetails table as shown below
+``` 
++---------+--------------+------+-----+---------+-------+
+| Field   | Type         | Null | Key | Default | Extra |
++---------+--------------+------+-----+---------+-------+
+| id      | int(11)      | NO   | PRI | NULL    |       |
+| name    | varchar(255) | YES  |     | NULL    |       |
+| address | varchar(255) | YES  |     | NULL    |       |
+| gender  | varchar(255) | YES  |     | NULL    |       |
++---------+--------------+------+-----+---------+-------+
+```
+The  StudentResultsDB database is used to store the student's results data in the StudentResults as shown below
+
+``` 
++-----------+------------+------+-----+---------+-------+
+| Field     | Type       | Null | Key | Default | Extra |
++-----------+------------+------+-----+---------+-------+
+| ID        | int(11)    | NO   | PRI | NULL    |       |
+| Com_Maths | varchar(1) | YES  |     | NULL    |       |
+| Physics   | varchar(1) | YES  |     | NULL    |       |
+| Chemistry | varchar(1) | YES  |     | NULL    |       |
++-----------+------------+------+-----+---------+-------+
+```
+The student's data is added through the request in the following service. However, the student's result data must be located in the database as shown below.
+```
++-----+-----------+---------+-----------+
+| ID  | Com_Maths | Physics | Chemistry |
++-----+-----------+---------+-----------+
+| 100 | A         | A       | A         |
+| 101 | A         | A       | B         |
+| 102 | A         | A       | C         |
+| 103 | A         | B       | A         |
+| 104 | A         | B       | B         |
+| 105 | A         | B       | C         |
+| 106 | A         | C       | A         |
+| 107 | A         | C       | B         |
+| 108 | A         | C       | C         |
++-----+-----------+---------+-----------+
+```
+
+### Developing the service
+
+To implement the scenario, let's start by implementing the message_transformation.bal file, which is the main file in the implementation. This file includes 4 main services as contentfilter, validate, enricher, and backend. Refer to the code attached below. Inline comments are added for better understanding.
+
+> In the below service you have to change below configurations according to your mysql configuration details, 
```

Comment:
Change 'mysql' to 'MySQL'. Please make this change everywhere in this guide.

Reply:
Done